Directory.Build.props

```diff
@@ -1,13 +1,15 @@
 <Project>
 
   <PropertyGroup>
-    <Version>3.9.4</Version>
+    <Version>3.10.0-alpha.0</Version>
```

Comment:
Setting a suffix so we can refer to preview dependencies.

Technically not bad to leave it by default since `craft` bumps to a new version on reach release. So this would make sure we can tell apart data in Sentry that came from a proper release vs some test branch we used for testing.

Reply:
likely needs a 'post-release.sh' script to append an `alpha.0` back on after each craft release